### DISCUSSION TITLE
How to create a table directly using the schema from an ibis dataframe?

### DISCUSSION THREAD

**Discussion by stereoF** [ibis-project/ibis#8911] [Q&A]
How to create a table directly using the schema from an ibis dataframe?

I've built a dataframe with ibis: df_play. I wanted to directly use df_play's schema to create a table in the database, but it failed:
`con2.create_table('algo_play', schema=df_play.schema())
`

Error message:
DatabaseError: HTTPDriver for http://cc-j6c9l15edn66o817k.public.clickhouse.ads.aliyuncs.com:8123/ returned response code 400) Code: 62. DB::Exception: Syntax error: failed at position 620 (end of query) (line 19, col 12): . Expected one of: token, Arrow, DoubleColon, non-empty parenthesized list of expressions, end of query. (SYNTAX_ERROR) (version 22.8.5.29)

**Comment by cpcloud** [comment]
Can you show the schema of `df_play.schema()`?

**Reply by cpcloud** [reply]
Actually, please open a bug report for this, thanks!
